Fix incomplete setutxent/endutxent defines for when there's no utmpx
Upstream bug #551

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura sudo-1.8.4p4.orig/src/utmp.c sudo-1.8.4p4/src/utmp.c
--- sudo-1.8.4p4.orig/src/utmp.c	2012-03-30 07:10:38.381706315 -0300
+++ sudo-1.8.4p4/src/utmp.c	2012-03-30 07:10:52.605802501 -0300
@@ -63,8 +63,8 @@
 #if !defined(HAVE_GETUTXID) && defined(HAVE_GETUTID)
 # define getutxline(u)	getutline(u)
 # define pututxline(u)	pututline(u)
-# define setutxent	setutent(u)
-# define endutxent	endutent(u)
+# define setutxent(u)	setutent(u)
+# define endutxent(u)	endutent(u)
 #endif /* !HAVE_GETUTXID && HAVE_GETUTID */
 
 #ifdef HAVE_GETUTXID
